### PR TITLE
test: admin login

### PR DESCRIPTION
**Pull request by FRSgit**

### CODE REVIEW

File: tests/frschool-e2e/test-utils/utils.ts [hunk 1]
```diff
@@ -1,13 +1,69 @@
-import { Page } from '@playwright/test';
+import test, { Page, expect } from "@playwright/test";
+
+export const AUTH_FILE_PATH = "playwright/.auth/user.json" as const;
+export const AUTH_ADMIN_FILE_PATH = "playwright/.auth/admin.json" as const;
+
+export const testAsRole = (role: "user" | "admin" | "guest") => {
```
Comment (MaciejRadym) [inline tests/frschool-e2e/test-utils/utils.ts:6]: how about 'useRole' like in react? :D

**Review by MaciejRadym — APPROVED** [review]

File: tests/frschool-e2e/test-utils/utils.ts (hunk 1 above)
Comment (MaciejRadym) [reply tests/frschool-e2e/test-utils/utils.ts:6]: `setTestRole` would also be accurate

File: tests/frschool-e2e/tests/auth.setup.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+test("has login prompt", async ({ page }) => {
+  await skipIfAuthCookieValid(AUTH_FILE_PATH);
+
+  await page.goto("/");
+
+  await isNuxtHydrated(page);
+
+  await page.getByTestId("pre-auth-login-button").click();
+  await expect(page.getByTestId("login-prompt")).toContainText("Zaloguj się!");
```
Comment (Doggnito) [inline tests/frschool-e2e/tests/auth.setup.ts:36]: it could be selected in other way than in toContainByText

**Review by Doggnito — APPROVED** [review]

File: tests/frschool-e2e/test-utils/utils.ts [hunk 3]
```diff
@@ -1,13 +1,69 @@
...
+  page,
+  login,
+  password,
+}: {
+  page: Page;
+  login: string;
+  password: string;
+}) => {
+  await page.getByTestId("login-form-email").clear();
+  await page.getByTestId("login-form-email").pressSequentially(login);
```
[the remark this answers is not in the document]
Comment (FRSgit) [reply tests/frschool-e2e/test-utils/utils.ts:38]: that's true, but we will come with better structure for our tests in next sessions - for now I'm keeping it simple 😄

File: tests/frschool-e2e/test-utils/utils.ts (hunk 1 above)
Comment (FRSgit) [reply tests/frschool-e2e/test-utils/utils.ts]: 6882a4cd6c848b8ccce1791d5215f280a29e241c

**Pull request merged**